Nyx92/3D-Model-Ecommerce-Frontend
# Fix/stripe payments improvements/stacey

## Changes
- fixed bug with Stripe checkout not resolving card data correctly
- added colour sample to item preview page
- added feature to remove cart cookies when SuccessCheckout page is loaded

## How to test
- Place an order for Clicky and ensure that 1) test card number resolves successfully, and 2) cart is emptied on payment success page

## Known Issues
- order data is not posted to DB successfully - SuccessCheckout page will reflect the order number as 0000

## Screenshots (if any)
Sample of colour preview:
![image](https://user-images.githubusercontent.com/47405262/218316938-651891d8-936a-4ce1-a3ab-f52ce1225390.png)
